app/models/federal_register_api.rb

```diff
@@ -0,0 +1,97 @@
+class FederalRegisterApi
+  def self.find_sorns
+    # Makes queries to the Federal Register API to find SORNs.
+    # Searching for 'Privacy Act of 1974; System of Records' of type 'Notice' is the best query we have found.
+    # In the results we still filter on those with a title that includes 'Privacy Act of 1974'.
+
+    conditions = { term: 'Privacy Act of 1974; System of Records' }#, agencies: ['general-services-administration'] }
```

Comment:
would it make sense to put the `conditions`, `fields` and `search options` in an class constructor?

Reply:
Changed this now use instances of the class. It'll be easier to make passing in variables now, if we ever need to add that ability.